packages/react-bootstrap-table2/src/bootstrap-table.js

```diff
@@ -164,7 +166,8 @@ BootstrapTable.propTypes = {
   onTableChange: PropTypes.func,
   onSort: PropTypes.func,
   onFilter: PropTypes.func,
-  onExternalFilter: PropTypes.func
+  onExternalFilter: PropTypes.func,
+  sortCaret: PropTypes.func
```

Comment:
Please help to rename `sortCaret` to `sortCaretRenderer`

Reply:
yeah thats a better name